v2/python/README.md

```diff
@@ -0,0 +1,54 @@
+# AdSense Management API v2 Samples
+
+A collection of command-line samples for the AdSense Management API.
+
+##Installation and first request
+
+1. Download Google APIs Client Library for Python (google-api-python-client) and
+   the OAuth library
+   (https://github.com/googleapis/google-auth-library-python-oauthlib):
+  - https://code.google.com/p/google-api-python-client/
+  - https://github.com/googleapis/google-auth-library-python-oauthlib
+
+  or use pip:
+
+  ```bash
+  $ pip install google-api-python-client
+  $ pip install google_auth_oauthlib
+  ```
+
+2. (No longer needed, check on this) Make sure you can import the client library:
```

Comment:
Just follow up on if this step is needed.

Reply:
Ah yes, the lingering TODO. Actually, this step still makes sense. I updated the name of the client library in my latest commit.